src/Microsoft.Health.Dicom.SqlServer/DicomSqlServerResource.resx

```diff
@@ -117,6 +117,10 @@
   <resheader name="writer">
     <value>System.Resources.ResXResourceWriter, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
   </resheader>
+  <data name="InvalidCustomTagStatus" xml:space="preserve">
+    <value>The custom tag '{0}' is not in valid status.</value>
```

Comment:
Perhaps something a bit more specific. As a user I'd say, "The heck does that mean?" lol

Reply:
how about "The custom tag '{0}' is being processed, please retry later."

Reply:
will change to "The custom tag '{0}' is being processed"